images/github-runner/Dockerfile

```diff
@@ -0,0 +1,50 @@
+FROM ubuntu:latest
+
+
+# system update
+RUN set -eux ; \
+    apt update ; \
+    apt -y upgrade ; \
+    apt -y install \
+            curl \
+            libicu-dev  \
+            build-essential \
+            git \
+            cmake \
+            python3-pip \
+            wget \
+            language-pack-en \
+            libcurl4-openssl-dev;
+
+# cuda install
+# https://docs.nvidia.com/cuda/cuda-installation-guide-linux/index.html
+ENV DEBIAN_FRONTEND=noninteractive
+RUN set -eux ; \
```

Comment:
Maybe here we should use the official nvidia image, but I prefer this approach to control the bench environment.